packages/components/src/Chip/Chip.types.d.ts

```diff
@@ -0,0 +1,34 @@
+import React, { PropsWithChildren } from "react";
+
+export interface ChipProps extends PropsWithChildren {
+  /** Accessible Label */
+  ariaLabel?: string;
+  /** Disables Chip */
+  disabled?: boolean;
+  /** Adds more prominent text to act as a heading */
+  heading?: string;
+  /** Changes Chip styling to inform the user of an issue. */
+  invalid?: boolean;
+  /** What do you want Chip to say? */
+  label: string;
+  /** What role does Chip play? */
+  role?: string;
+  /** Defaults to zero. Accessible feature. */
```

Comment:
I think we could make these descriptions more descriptive. I don't think we have any components using this structure of asking questions to the user. Also, when there's a default value, we can also expose here to the consumer

Example: https://github.com/GetJobber/atlantis/blob/e0183893629b508a397c8a41cb0aed08121db314/packages/components/src/Avatar/Avatar.tsx

Reply:
Done. I've tried my best to be as descriptive as possible without any cheekyness, and pulled the variations out into an exported type.